frontend/src/app/forms/side-menu/side-menu.component.scss

```diff
@@ -0,0 +1,3 @@
+.font25 {
```

Comment:
The same as for the forms.page.scss

Reply:
Ya it is temporary change. Once a definite way to handle CSS , this will be changed.